src/manifest/clog.sidecar

```diff
@@ -22907,6 +22907,34 @@
         },
         "CustomSettings": null
       }
+    },
+    "PerfCountersRundown": {
+      "ModuleProperites": {},
+      "TraceString": "[ lib] Perf counters Rundown, Counters=%!CID!",
+      "UniqueId": "PerfCountersRundown",
+      "splitArgs": [
+        {
+          "VariableInfo": {
+            "UserSpecifiedUnModified": "\r\n            CLOG_BYTEARRAY(sizeof(PerfCounters), PerfCounters)",
```

Comment:
I don't think this line is necessary any longer. Can you update your clog and delete this and see if things still build?